packages/features/bookings/lib/EventManager.ts

```diff
@@ -388,13 +389,16 @@ export default class EventManager {
         thirdPartyRecurringEventId = result.createdEvent?.thirdPartyRecurringEventId;
       }
 
+      const createdEvent = result.createdEvent as { providerUrl?: string; url?: string } | undefined;
```

Comment:
Instead of this type assertion, can we improve the type of createdEvent itself.

Type assertions are bad for runtime code.